### ISSUE TITLE
How to generate topics

### ISSUE THREAD

**Issue by dangpnh2** [Dan123dan/rGBN-RNN#1]
Hi,
How do I generate topics and their hierarchy from your trained model?

**Comment by Dan123dan** [comment]
Yeah. The hierarchical topics $Phi^{1:L}$ are learned from the topic model. And taking the Phi^{1} at the example, it is a matrix with the size of V*K_1, where  K_1 is the number of topics at layer 1 and $V$ is the size of the vocabulary of the topic model. To visualize the topics k at layer 1 (k \in K_1),  i.e., the k-th column of \Phi^{1}, you can list the representative words in the V-dimensional vector with decreasing ranks.
